django/utils/crypto.py

```diff
@@ -87,3 +87,11 @@ def pbkdf2(password, salt, iterations, dklen=0, digest=None):
     password = force_bytes(password)
     salt = force_bytes(salt)
     return hashlib.pbkdf2_hmac(digest().name, password, salt, iterations, dklen)
+
+
+def scrypt(password, salt, n=16384, r=8, p=1, maxmem=0, dklen=64):
```

Comment:
I do not think we need this function.

Reply:
The `scrypt ` function converts the string password and salt into bytes. Should I use the `force_bytes` method in the hashers.py scrypt class then? I just followed the format of the pbkdf2 function above the `scrypt` function I defined.

Reply:
I agree with Florian, it's enough to call `encode()`.